odoo/addons/base/models/ir_cron.py

```diff
@@ -393,36 +394,53 @@ def toggle(self, model, domain):
         return self.try_write({'active': active})
 
     @api.model
-    def _trigger(self, delay=None, at=None):
+    def _trigger(self, at=None):
         """
         Schedule a cron job to be executed soon independently of its
         ``nextcall`` field value.
 
         By default the cron is scheduled to be executed in the next
-        batch but optional arguments may be given to delay the execution
-        with a precision down to 1 minute.
-
-        :param datetime.timedelta delay:
-            Execute the cron later, after the delay expires. 
+        batch but the optional `at` argument may be given to delay the
+        execution later with a precision down to 1 minute.
 
         :param datetime.datetime at:
-            Execute the cron later, at a precise moment in time.
+            Execute the cron later, at a precise moment in time instead
+            of as soon as possible.
         """
         self.ensure_one()
+        self._trigger_multi([at or datetime.utcnow()])
+
+
+    @api.model
+    def _trigger_multi(self, at_list):
+        """
+        Schedule a cron job to be executed at every ``at_list`` time
+        independently of its ``nextcall`` field value.
 
-        if delay and at:
-            raise ValueError("Set either a delay either a moment, not both.")
+        :param list[datetime.datetime] at_list:
+            Execute the cron later, at precise moments in time.
+        """
 
-        now = datetime.utcnow()
-        call_at = at or now
-        if delay:
-            call_at += delay
+        if not at_list:
+            return
 
-        self.env['ir.cron.trigger'].sudo().create({'cron_id': self.id, 'call_at': call_at})
-        _logger.debug("will execute '%s' at %s",
-            self.name, call_at if call_at > now else "soon as possible")
+        # compress and sort the list to insert at most one trigger per minute
+        at_list = [
+            group
+            for group, originals
+            in itertools.groupby(
+                sorted(at_list),
+                functools.partial(datetime.replace, second=0, microsecond=0)
+            )
+        ]
```

Comment:
You can alternatively round values to the minute, and then uniquify the list:
```suggestion
        # compress and sort the list to insert at most one trigger per minute
        at_list = unique(at.replace(second=0, microsecond=0) for at in at_list)
```